src/ripple/shamap/SHAMapItem.h

```diff
@@ -33,87 +34,58 @@ namespace ripple {
 class SHAMapItem
 {
 private:
-    uint256 mTag;
-    Serializer mData;
+    uint256    tag_;
```

Comment:
Code  with aligned columns 
might be    more     aesthetic
to       you  but        people
read   from  left       to 
right    not    up        and 
down!